libs/protocol-client/src/lib/graphql/queries.graphql

```diff
@@ -595,9 +595,17 @@ fragment ContributionFragment on Contribution {
   guilds {
     id
     guild_id
+    attestation_threshold
+    verification_status_id
+    verified
     guild {
       id
       name
+      verification_setting_id
```

Comment:
Isn't verification_setting_id a duplicate of verification_setting.id

Reply:
Ah, yes. I added this in before we switched that over

Reply:
If this change is in #465 I'll fix the conflicts after that's in

Reply:
I'll need to change this in a few other places as well outside of this PR